mautic/mautic
# Disable trackable urls in SMS integration

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | ✅ 
| Automated tests included? | ✅ 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/2357 & #1873
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
PR added disable trackable urls in SMS. 
Mautic has very long trackable URL, this option allow use your own urls.

#### Steps to test this PR:
1.  Apply PR, setup Twillio integration and enable Disable trackable url option
2.  Create SMS with url
3. Send SMS through campaign action and see if your URL wasn't replace Mautic trackable url
